tuf/signed/interface.go

```diff
@@ -18,6 +19,9 @@ type KeyService interface {
 	// GetKey retrieves the public key if present, otherwise it returns nil
 	GetKey(keyID string) data.PublicKey
 
+	// GetKeyInfo retrieves the KeyInfo for a keyID
+	GetKeyInfo(keyID string) (trustmanager.KeyInfo, error)
```

Comment:
Group review: seems like we don't need this, can it be deleted from the interface and implementations?